--- src/profiler.cc	2013-09-03 21:35:21.000000000 +0800
+++ src/profiler_new.cc	2013-09-03 21:36:06.000000000 +0800
@@ -205,7 +205,7 @@
   if (signal_number_str != NULL) {
     long int signal_number = strtol(signal_number_str, NULL, 10);
     if (signal_number >= 1 && signal_number <= 64) {
-      void *old_signal_handler = reinterpret_cast<void *>(signal(signal_number, CpuProfilerSwitch));
+      void *old_signal_handler = (void *)(signal(signal_number, CpuProfilerSwitch));
       if (old_signal_handler == NULL) {
       	RAW_LOG(INFO,"Using signal %d as cpu profiling switch", signal_number);
       } else {
